### ISSUE TITLE
Help regarding dataset and code [Urgent]

### ISSUE THREAD

**Issue by rohanalgo** [xthan/polyvore-dataset#6]
I want to make a model around fashion compatibility such that it predicts the best outfit combination.

The dataset on kaggle have only images, and i could not find a way to map it with tar file json information. I dont think , as per the paper, its gonna work without the mapping.

Is there a way to make it work ?

**Comment by xthan** [comment]
The tid in the broken image url should match the image file name on Kaggle.

**Comment by rohanalgo** [comment]
yes the set id matches. thank you. But will the code directly work or i have to replace the existing image url ?
